source/clear-linux/guides/telemetrics/telem-guide.rst

```diff
@@ -358,12 +360,12 @@ using the :ref:`bare-metal-install-server` getting started guide and:
       passwd: password updated successfully
 
    Enter `postgres` for the current value of the password and then enter a new
-   password, retype it to verify the new password and the
-   :guilabel:`PostgreSQL` database password will be updated.
+   password, retype it to verify the new password and the :guilabel:`PostgreSQL`
+   database password will be updated.
 
 #. After the installation is complete, you can use your web browser to view the
-   new server by opening the browser on the system and typing in ``localhost``
-   in the address bar.  You should see a web page similar to the one shown in
+   new server by opening the browser on the system and typing in localhost
+   in the address bar. You should see a web page similar to the one shown in
    figure 1:
```

Comment:
"Figure 2 below."   (cap F, 2, no colon)  
Also change caption to "Figure 2" in line 374.